src/odin/ingestion/afc/afc_archive.py

```diff
@@ -68,12 +79,68 @@ def make_pl_schema(schema_list: ApiSchema) -> pl.Schema:
         # "timestamp with time zone": pl.Datetime(), # Polars can't automatically parse ts from api
     }
     r_schema = {}
-    for schema_d in schema_list:
-        r_schema[schema_d["column_name"]] = converter.get(schema_d["data_type"], pl.String())
+    for col in schema["table_infos"]:
+        r_schema[col["column_name"]] = converter.get(col["data_type"], pl.String())
 
     return pl.Schema(r_schema)
 
 
+def verify_downloads(csv_path: str, csv_schema: pl.Schema, download_jobs: APICounts) -> None:
```

Comment:
Not really needed as class function, so moved to external function and added testing. But no change to any code in here.